.github/workflows/test.yaml

```diff
@@ -52,13 +59,58 @@ jobs:
         run: cat ./axelar-chains-config/info/local.json
 
       - name: Deploy ConstAddressDeployer
-        run: node evm/deploy-contract.js -a ../node_modules/@axelar-network/axelar-gmp-sdk-solidity/artifacts/contracts/deploy/ -c ConstAddressDeployer -m create -y
+        run: node evm/deploy-contract.js -c ConstAddressDeployer -m create -y
 
       - name: Deploy Create3Deployer
-        run: node evm/deploy-contract.js -a ../node_modules/@axelar-network/axelar-gmp-sdk-solidity/artifacts/contracts/deploy/ -c Create3Deployer -m create2 -y
+        run: node evm/deploy-contract.js -c Create3Deployer -m create2 -y
 
       - name: Deploy AxelarGateway
         run: node evm/deploy-gateway-v6.2.x.js -m create3 -s "AxelarGateway v6.2" -g 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 -m 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 --keyID 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 -y
 
       - name: Call Method on Gateway
         run: node evm/gateway.js --action callContract --destinationChain test2 -y
+
+      - name: Deploy InterchainGovernance using create
+        run: node evm/deploy-contract.js -c InterchainGovernance -m create -y
+
+      - name: Deploy AxelarServiceGovernance using create
+        run: node evm/deploy-contract.js -c AxelarServiceGovernance -m create -y
+
+      - name: Deploy Multisig using create
+        run: node evm/deploy-contract.js -c Multisig -m create -y
+
+      - name: Deploy Operators using create
+        run: node evm/deploy-contract.js -c Operators -m create -y
+
+      - name: Deploy TokenDeployer using create
+        run: node evm/deploy-contract.js -c TokenDeployer -m create -y
+
+      - name: Deploy InterchainGovernance using create2
+        run: node evm/deploy-contract.js -c InterchainGovernance -m create2 -y
+
+      - name: Deploy AxelarServiceGovernance using create2
+        run: node evm/deploy-contract.js -c AxelarServiceGovernance -m create2 -y
+
+      - name: Deploy Multisig using create2
+        run: node evm/deploy-contract.js -c Multisig -m create2 -y
+
+      - name: Deploy Operators using create2
```

Comment:
i'd keep each with create3, and just one with create/create2 to not increase the gh action time too much